src/app/_types/sharedProps/imageType.ts

```diff
@@ -1,4 +1,9 @@
+import { StaticImageData, ImageProps as NextImageProps } from 'next/image'
+
 export type ImageProps = {
   url: string
-  alt: string
-}
+} & Omit<NextImageProps, 'src' | 'objectFit'>
```

Comment:
Are we omitting this because we have `url` instead of `src`?

Reply:
Exactly!

`url` is for `ImageProps` and `data` is for `StaticImageProps`

Reply:
Let's create a ticket to change `url` to `src` and remove this omission. 🙏🏼